README.md

```diff
@@ -1,4 +1,7 @@
 # trading_calendars
+
+[![Build Status](https://travis-ci.org/quantopian/trading_calendars.svg?branch=master)](https://travis-ci.org/quantopian/trading_calendars)
```

Comment:
I'm not familiar with the convention of including this in a README, is this standard? (disclaimer, I'm not an engineer :) )

Reply:
Yep, this is the Markdown syntax for a badge like the ones at the top of the Zipline README: https://github.com/quantopian/zipline/blob/master/README.rst